libs/hmpb/src/all_bubble_ballot_fixtures.tsx

```diff
@@ -176,6 +180,8 @@ function BallotPageFrame({
             precinctId={election.precincts[0].id}
             pageNumber={pageNumber}
             totalPages={totalPages}
+            condenseFooterMetadata
+            includeCustomFooterMetadataSlot
```

Comment:
I'm a little hesitant to add these custom props to the footer element. It feels a little overcomplicated.

Two ideas for other ways to approach this:
1. If we want the all bubble ballot to have the same footer as the default ballot template, then we might consider having one less row of bubbles in the all bubble ballot template. Did you look into removing a row of bubbles (and updating the election definition accordingly)?
2. Since there's this new cycle index field that you added, maybe the all bubble ballot could have its own footer component that meets its specific needs (QR code, ballot hash, and cycle index).

Reply:
Agreed that current params result in an abstraction that doesn't quite make sense. I like the idea of a unique footer component for the all-bubble ballot!

I did consider removing a row of bubbles but was looking for a quick solution last week and rearranging the footer proved to be quite simple. Happy to look into that now for a more permanent fixture/solution